.rubocop_todo.yml

```diff
@@ -971,6 +971,7 @@ Style/PerlBackrefs:
 Style/PredicateName:
   Exclude:
     - 'spec/**/*'
+    - 'src/api/app/controllers/webui/kiwi/images_controller.rb'
```

Comment:
Just clarified with Ana. She explained it to me why she thinks it's useful and I agree now. So I'm fine with it.